broker/cloud_functions/check_cue_response/deployment.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+# Deploys or deletes broker Cloud Function
+# This script will not delete Cloud Functions that are in production
+
+testid="${1:-test}"
+# "False" uses production resources
+# any other string will be appended to the names of all resources
+teardown="${2:-False}"
+# "True" tearsdown/deletes resources, else setup
+survey="${3:-ztf}"
+# name of the survey this broker instance will ingest
+versiontag="${4:-v3_3}"
+zone="${CE_ZONE:-us-central1-a}" # use env variable CE_ZONE if it exists
```

Comment:
```suggestion
zone="${5:-us-central1-a}"
```

As of #192, we no longer use the `CE_ZONE` environment variable, so let's change it to `5` so that the script accepts it as an input argument. If the variable isn't actually being used in the script (as Codacy indicates here, and in other files below) it should just be removed altogether.

Reply:
I reviewed the bash scripts (`deploy.sh`) and python scripts (`main.py`) for each of the individual cloud functions. It seems that `check_cue_response` is the only cloud function which uses `zone`. Previously, `zone` is used in `main.py`, but not `deploy.sh`. I've since added `zone` as an environment variable that gets passed when the command `gcloud functions deploy "${check_cue_CF_name}" \` is run in `deploy.sh`